cmake/projects/Avahi/hunter.cmake

```diff
@@ -20,10 +21,11 @@ hunter_add_version(
 )
 
 hunter_cmake_args(
-    Avahi 
+    Avahi
     CMAKE_ARGS
-      BUILD_SHARED_LIBS=ON  
+      BUILD_SHARED_LIBS=ON
 )
 
+hunter_configuration_types(Avahi CONFIGURATION_TYPES Release)
 hunter_pick_scheme(DEFAULT url_sha1_avahi_autotools)
 hunter_download(PACKAGE_NAME Avahi)
```

Comment:
I'm afraid that since internal structure of scheme changed we have to add `PACKAGE_INTERNAL_DEPS_ID`. See https://github.com/ruslo/hunter/wiki/dev.modules#hunter_download. Problem will occur in case somebody already save this package in cache by adding `HUNTER_CONFIGURATION_TYPES=Release`. It means if somebody build all 3rd parties with `Release` then `Avahi` saved to cache without new flags. Now when we add flags cache system doesn't see updates because think that: source doesn't change + build type doesn't change (Release was global, now it's local default but still Release too) + internal deps id doesn't change => using version from cache (without flags).